setup.py

```diff
@@ -45,7 +43,7 @@
 
 setup(
     name='iotedgedev',
-    version='2.2.0',
+    version='3.0.0',
```

Comment:
>version='3.0.0 [](start = 4, length = 14)

same comment about releasing an rc version first. The OneNote has isntructions for all the places in the code that need to be updated for the RC version release to go through.